app/lib/actions/pulp3/orchestration/capsule_content/remove_orphans.rb

```diff
@@ -0,0 +1,14 @@
+module Actions
+  module Pulp3
+    module Orchestration
+      module CapsuleContent
+        class RemoveOrphans < Pulp::Abstract
+          def plan(proxy)
+            plan_action(Actions::Pulp3::Orchestration::Repository::RemoveOrphans, proxy) if proxy == SmartProxy.pulp_master
+            plan_action(Actions::Pulp3::CapsuleContent::RemoveUnneededRepos, proxy) if proxy.pulp_mirror?
```

Comment:
I think some of the action naming is getting confusing.  Lets try to come up with a structure for everything we need, something like:
```
Actions::Katello::CapsuleContent::OrphanCleanup
   Actions::Pulp::Orchestration::CaspuleContent::OrphanCleanup
     Actions::Pulp::CapsuleContent::RemoveUnneededRepositories
     Actions::Pulp::CapsuleContent::RemoveOrphanContentUnits
   Actions::Pulp3::Orchestration::CaspuleContent::OrphanCleanup
         Actions::Pulp3::CaspuleContent::RemoveOrphanedRepositories
         Actions::Pulp3::CaspuleContent::RemoveOrphanedRepositoryVersions   #here i'd probably do the if/else check for master or mirror
         Actions::Pulp3::CaspuleContent::RemoveOrphanedDistributions      #future work
         Actions::Pulp3::CaspuleContent::RemoveOrphanedContentUnits     #future work

```

And maybe instead of a 'CapsuleContent' namespace, lets move it to  'Actions::*::OrphanCleanup::'  I'd like to see all capsule content references eventually go away

Reply:
Thoughts?

Reply:
So it might look like:
`Actions::Katello::CapsuleContent::OrphanCleanup `-> `Actions::Katello::OrphanCleanup`

Reply:
yep